assets/pages/new-contributions-landing/setUserStateActions.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import { defaultUserActionFunctions } from 'helpers/user/defaultUserActionFunctions';
+import { setFormSubmissionDependentValue } from './checkoutFormIsSubmittableActions';
+
+// ----- Actions Creators ----- //
+
+const setIsSignedIn = (isSignedIn: boolean): ((Function) => void) =>
+  (dispatch: Function): void => {
+    dispatch(setFormSubmissionDependentValue(() => ({ type: 'SET_IS_SIGNED_IN', isSignedIn })));
+  };
+
+const setIsRecurringContributor = (): ((Function) => void) =>
+  (dispatch: Function): void => {
+    dispatch(setFormSubmissionDependentValue(() => ({ type: 'SET_IS_RECURRING_CONTRIBUTOR' })));
+  };
+
+const setUserStateActions = {
```

Comment:
re: the exact type (my gut was telling me it was a good idea but I was trying to think of a specific example!). here, if you had a typo in your override property, it would type-check if it wasn't an exact type. but with the exact type, flow would pick this up